module_0_introduction/intro_to_python.ipynb

```diff
@@ -18,7 +18,8 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Please note that we are using Python 3 (__NOT__ Python 2! Python 2 has some different functions and syntax)."
+    "Please note that we are using Python 3.    \n",
```

Comment:
We need to make a decision on which specific python 3 version we are using

Reply:
My vote is the current version which is 3.6 until 6/2018 then it will 3.7.

The minor releases don't matter since they are mostly security patches.